app/customer.py

```diff
@@ -0,0 +1,53 @@
+from __future__ import annotations
+
+from dataclasses import dataclass
+from math import sqrt
+
+from app.shop import Shop
+from app.user_data import FUEL_PRICE
+
+
+@dataclass
+class Customer:
+    name: str
+    product_cart: dict
+    location: list
+    money: int | float
+    car: dict
+
+    def get_money(self) -> None:
+        print(f"{self.name} has {self.money} dollars")
+
+    def get_distance(self, other: Shop) -> float:
+        return (
+            round((sqrt((other.location[0]
+                         - self.location[0]) ** 2
+                        + (other.location[1]
+                           - self.location[1]) ** 2)) * 2, 2)
+        )
```

Comment:
consider breaking down the expression into smaller, more descriptive variables to enhance clarity.